src/Workspaces/Core/Portable/Workspace/ProjectSystem/ProjectSystemProject.cs

```diff
@@ -1030,6 +1030,49 @@ public void RemoveAnalyzerReference(string fullPath)
         }
     }
 
+    private OneOrMany<string> GetMappedAnalyzerPaths(string fullPath)
+    {
+        fullPath = Path.GetFullPath(fullPath);
+
+        if (IsSdkCodeStyleAnalyzer(fullPath))
+        {
+            // We discard the CodeStyle analyzers added by the SDK when the EnforceCodeStyleInBuild property is set.
+            // The same analyzers ship in-box as part of the Features layer and are version matched to the compiler.
+            return OneOrMany<string>.Empty;
```

Comment:
Will this work as is for VSCode?  I noticed that we don't have a codestyle dll shipped in the extension - do we need that, or do we just need the Features dll?

Reply:
This does work in VS Code. We only need the Features assemblies.